src/components/core/button/button.tsx

```diff
@@ -81,15 +84,30 @@ const Button = styled('button', {
         },
     },
     compoundVariants: [
+        {
+            color: 'primary',
+            dark: true,
+            css: {
+                backgroundColor: '$coral500',
+                color: '$greyLight100',
+
+                '&:hover': {
+                    backgroundColor: '$coral400',
+                },
+                '&:disabled': {
+                    opacity: 0.32,
+                },
+            },
+        },
         {
             color: 'primary-light',
             dark: true,
             css: {
-                backgroundColor: 'rgba(255, 68, 79, 0.16)',
-                color: '#ff444f',
+                backgroundColor: '$coral500_16',
+                color: '$coral500',
 
                 '&:hover': {
-                    backgroundColor: 'rgba(255, 68, 79, 0.24)',
+                    backgroundColor: '$coral500_24',
                 },
                 '&:disabled': {
                     opacity: 0.32,
```

Comment:
should it be just $32? 😊
```suggestion
                    opacity: '$32',
```